### PR TITLE
Migrate to ghcr

### PR DESCRIPTION
**Pull request by francoisferrand**
- Use backbeat from ghcr
- Use docker/login-action to setup registry
- Migrate to ghcr
- Upgrade actions
- Use cloudserver from ghcr
- Use ctst from ghcr
- Use kafka-cleaner from ghcr
- Use sorbet from ghcr
- Add sorbet dashboards
- Use shell-ui from ghcr
- Use pensieve-api from ghcr
- Use s3utils from ghcr
- Use static container registry from ghcr
- Use utapi from ghcr
- Use vault from ghcr
- Use zkop from ghcr
- Use zenko-ui from ghcr

Issue: ZENKO-4800

### CODE REVIEW

**Review by ghost — APPROVED** [review]
:tada:

File: tests/ctst/README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ folders. The image can be built and pushed with the following steps:
 cd ./tests/ctst/
 
 # Building the image
-docker build --build-arg CTST_TAG=0.2.0 . -t registry.scality.com/playground/<username>/custom-ctst:0.2.0
+docker build --build-arg CTST_TAG=0.2.0 . -t ghcr.io/scality/playground/<username>/custom-ctst:0.2.0
 
 # Pushing the custom image into a repository
-docker push registry.scality.com/playground/<username>/custom-ctst:0.2.0
+docker push ghcr.io/scality/playground/<username>/custom-ctst:0.2.0
```
Comment (ghost) [inline tests/ctst/README.md:26]: Will this work with ghcr?

Comment (francoisferrand) [reply tests/ctst/README.md:26]: it should, according to Thomas...

**Review by BourgoisMickael — APPROVED** [review]

**Review by KillianG — APPROVED** [review]

**Pull request merged**